packages/design-picker/src/components/index.tsx

```diff
@@ -168,7 +177,7 @@ const DesignButtonContainer: React.FC< DesignButtonContainerProps > = ( {
 	const isDesktop = useViewportMatch( 'large' );
 	const isBlankCanvas = isBlankCanvasDesign( props.design );
 
-	if ( ! onPreview ) {
+	if ( ! onPreview || props.hideFullScreenPreview ) {
 		return (
 			<div className="design-button-container">
```

Comment:
I think we can just add class name here to remove the pseudo element. Then, we don't need `hideFullScreenPreview` prop. Also, we can keep the border color of the pseudo element.
```css
.design-button-container--without-preview {
  .design-picker__image-frame::after {
    pointer-events: none;
  }

  &:hover,
  &:focus-within {
    .design-picker__image-frame::after {
      background-color: transparent;
    }
  }
}
```

Reply:
Hey, thanks for your feedback! I have made those changes, can you please have a look?

Reply:
Looks good!